### DISCUSSION TITLE
CODE_OF_CONDUCT.rst not being recognized by github as code of conduct

### DISCUSSION THREAD

**Discussion by aucampia** [community/community#9442] [Other Feature Feedback, Questions, & Ideas]
For some reason repositories with `CODE_OF_CONDUCT.rst` still report that there is no code of conduct in the community tab, eg: 

- https://github.com/InstituteforDiseaseModeling/covasim/blob/master/CODE_OF_CONDUCT.rst
- https://github.com/InstituteforDiseaseModeling/covasim/community
  ![image](https://user-images.githubusercontent.com/4481387/147784747-9ea7d61a-8dab-49e3-a864-4e97c15430d8.png)

The same problem was reported here: https://github.community/t/code-of-conduct-is-present-in-repo-but-insights-community-tab-doesnt-detect-it/1534/5

**Comment by jab** [comment]
I just found my way here from a Google search when I noticed this bug biting [my project](https://github.com/jab/bidict/community). Apparently I first noticed this so long ago that I forgot I even [posted about this in the community thread](https://github.community/t/code-of-conduct-is-present-in-repo-but-insights-community-tab-doesnt-detect-it/1534/7) a couple summers ago, and only saw that again thanks to your linking to it.

Thanks for posting this here, @aucampia, which is hopefully a better place for it to get some attention from the GitHub team. And hopefully it's as quick a fix as it seems like it might be to at least just recognize CODE_OF_CONDUCT.rst files as well as CODE_OF_CONDUCT.md files.

**Comment by jlaehne** [comment]
It works for `CONTRIBUTING.rst` files, so weird that it should not work for the code of conduct as well.

**Comment by Bibo-Joshi** [comment]
IISC the main issues is that GH 

> The tool will only check specifically for code of conduct files that we recognize, not just the existence of a `CODE_OF_CONDUCT.md`.

(Quote from [here](https://github.community/t/code-of-conduct-is-present-in-repo-but-insights-community-tab-doesnt-detect-it/1534/2)).

What could help is GH providing the accepted templates in a rst format rather than just md.

**Comment by pmocz** [comment]
Checking in on this issue in 2025. Have there been any updates?
